posixlib/src/main/scala/scala/scalanative/posix/limits.scala

```diff
@@ -1,10 +1,10 @@
 package scala.scalanative
 package posix
 
-import scalanative.unsafe.{extern, name, CInt}
+import scalanative.unsafe.{CSize, extern, name}
 
 @extern
 object limits {
   @name("scalanative_path_max")
-  def PATH_MAX: CInt = extern
+  def PATH_MAX: CSize = extern
```

Comment:
This should be `CInt` as in the spec and you should convert to `CSize` with a `val` if needed.
```Scala
val PATH_MAX: CSize = limits.PATH_MAX.toUInt
```
Since you are using it in two places, I am not sure the best place to put it. Worst case just duplicate in the 2 files like you do the `1U`.